tests/api/ask_astro/chains/test_answer_questions.py

```diff
@@ -0,0 +1,9 @@
+from langchain.prompts import SystemMessagePromptTemplate
+
+
+def test_system_prompt_loading():
+    """Test if the system prompt is loaded correctly"""
+    with open("ask_astro/templates/combine_docs_chat_prompt.txt") as fd:
```

Comment:
I used to use https://pypi.org/project/pytest-regressions/ for this kind of testing. If we're not against this, I can do this after https://github.com/astronomer/ask-astro/issues/82. Non-blocking either